### DISCUSSION TITLE
Explanation for each feature?

### DISCUSSION THREAD

**Discussion by BarbzYHOOL** [Floorp-Projects/Floorp#500] [Feature Request]
There are many settings in about:preferences that I don't know about, for example:
- Wrap scrolling tabs at the edge
- workspaces
- Show the title of pinned tabs (I'm using vertical tabs, and I see no difference with or without it)
- Containers tabs?
- Use autoscrolling?
- Display the Find Bar at the top of the page
- Enable Rest Mode shortcut (F9)

I don't know if they are from firefox either.

**Comment by CutterKnife** [comment]
1) "Wrap scrolling tabs at the edge" is a setting that scrolls to the end and back to the first tab when "Switch tabs by scrolling with your mouse" is enabled.
2) Workspaces is a Floorp original feature. Please see the following unofficial article.
https://fs-un.super.site/en/workspaces
3) Vertical Tabs is a Floorp original feature, but it is experimental. (IMO, Vertical Tabs don’t need to hide tab titles)
https://fs-un.super.site/en/vertical-tabs
4) Container Tabs is a Firefox feature, but Private Container is Floorp exclusive.
https://support.mozilla.org/en-US/kb/containers
https://blog.ablaze.one/3520/2023-09-23/
5) Autoscrolling is a function that keeps scrolling at the same speed when you move the cursor while pressing the middle button.
6) "Display the Find Bar at the top of the page" is a setting for the position of the find bar (to search for words on the page).
7) Rest Mode is a function that temporarily hides the display of Floorp.

Since the developer seems to lack the resources to write an article, I will consider adding it to the unofficial support site instead.

**Reply by BarbzYHOOL** [reply]
thank you very much, and perfect english!

5) nice, I used to use this thing for tons of years and suddenly it disappeared. Now I can have it back
6) Okay. So is it because the default position sometimes hides part of the text? seems like a good idea to set it on
7) i just tried it, not sure what it did except unload all tabs and display a popup. and display blank page

I will read all the links you sent and come back if needed

PS: a small icon in the settings next to each Floorp native feature should be added

**Reply by CutterKnife** [reply]
6) The Find Bar appears at the bottom by default, but if the address bar is at the top, it is more reasonable to place it at the top as well.
7) That is what this feature is all about. This allows you to temporarily have nothing on your screen when you are away from your PC for a short period of time.

**Reply by BarbzYHOOL** [reply]
Here https://fs-un.super.site/en/workspaces it says "When you start the browser, a hexagonal icon appears in the upper left corner. This is the icon that manages your Workspaces.
By default, the "Default" Workspace is selected." but there is no hexagonal icon for me

**Reply by Ryosuke-Asano** [reply]
Workspaces not enabled by default.

**Reply by CutterKnife** [reply]
Sorry. That was a misleading explanation.
Edit: fixed

**Comment by BarbzYHOOL** [comment]
8) What is the button in the url bar called "Floorp system"? it displays a QR code?? what is it?

9) Where is the "Reader view" feature? or "Reader mode"? to read articles

**Reply by Ryosuke-Asano** [reply]
Floorp system is a core extension.

Generate QR code (v11.5.0 will be replaced by native component), Inject to Gesturefy and etc…

Reader mode is Firefox feature. Please see Mozilla support article.

**Reply by BarbzYHOOL** [reply]
Ok, It generates a QR code of the current page url? It's also written in japanese
Also Floorp system should be renamed to something like "Floorp QR" in my opinion. 

So there is no reader mode in Floorp??? I don't find the button (it automatically appears on firefox)

**Reply by Ryosuke-Asano** [reply]
Maybe your using design is affect to hide reader mode icon. Please try to use lepton or proton

Please references my before comment. I re-implemented the QRcode generate feature. It will available on 11.5.0. 
<img width="340" alt="image" src="https://github.com/Floorp-Projects/Floorp/assets/73892113/9bc7be2d-65aa-4d31-8116-2fb0163c07ba">

ps: generator will generate current page’s url QR-code

**Reply by BarbzYHOOL** [reply]
I'm not japanese, I don't speak it.

Ok so the reader mode button does not appear ONLY with the "Floorp FLuerial UI". It works with the other themes. Weird?

**Reply by Ryosuke-Asano** [reply]
Sorry. My website translater edit my comment. I rewrite my comment. My apologies.

**Reply by BarbzYHOOL** [reply]
I had translated it anyway. As I said, Reader mode is bugged with "Floorp FLuerial UI"
